libticalcs/trunk/tests/test_ticalcs_2.cc

```diff
@@ -27,12 +27,115 @@
 #include <config.h>
 #endif
 
-#include <ctype.h>
-#include <errno.h>
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
+#include <cctype>
+#include <cerrno>
+#include <cstdio>
+#include <cstdlib>
+#include <cstring>
+
+#ifdef __WIN32__
```

Comment:
I know about the asymmetrical compatibility between GPLv2 and BSD code, but a GPLv2(+) implementation would be better :)

Reply:
Do you have one in mind?

Reply:
Alternatively, conditionally use the vcpkg getopt / getopt-win32 where needed ?
https://github.com/microsoft/vcpkg/tree/master/ports/getopt

Reply:
Ah, yes maybe, I'll take a look.

Reply:
Works !